.circleci/config.yml

```diff
@@ -10,9 +10,11 @@ jobs:
       - run: git diff --exit-code
 
   test:
-    machine: true
+    machine:
+      image: ubuntu-2004:202101-01
```

Comment:
This was required because `make jsonnet-vendor` was not working with the default older image (`bingo` requires a relatively new version of Go).